containersQa/testlib.bash

```diff
@@ -116,19 +116,50 @@ function skipIfRhel7OsExecution() {
   fi
 }
 
+#
+# Create a user session, so podman/cgropus2 works properly.
+# This solves problems like: "sd-bus call: Interactive authentication required"
+#
+# 1496-STEP 5/8: RUN chown -R 1001:0 /tmp/src
+# 1497:error running container: from /usr/bin/crun creating container for [/bin/sh -c chown -R 1001:0 /tmp/src]: sd-bus call: Interactive authentication required.: Permission denied
+# 1498-: exit status 1
+# 1499:time="2025-06-25T13:14:21Z" level=error msg="did not get container create message from subprocess: EOF"
+#
+# See https://www.redhat.com/en/blog/sudo-rootless-podman
+# See https://github.com/containers/podman/issues/19556
+#
+function prepareUserSession() {
+  if [[ -z $USERNAME ]] ; then
+    echo "prepareUserSession: No username specified"
+    return
+  fi
+  echo "prepareUserSession for user \"$USERNAME\""
+  loginctl enable-linger $USERNAME 
```

Comment:
The only thing which worries me is why it worked up to now

Reply:
Hmm. But the only case where I found sudo, is el7 launcher. Is it possible? If so, then maybe the   loginctl enable-linger should be on el7?  Or maybe the podman on el7 is finally correctly rootless, and sudo may be dropped?

Reply:
@zzambers  wdyt?